tests/30-main.spec.js

```diff
@@ -254,4 +255,448 @@ describe('main', () => {
+          VC_RL_CONTEXT_URL
+        ],
+        id: 'urn:uuid:a0418a78-7924-11ea-8a23-10bf48838a41',
+        type: ['VerifiableCredential', 'example:TestCredential'],
+        credentialSubject: {
+          id: 'urn:uuid:4886029a-7925-11ea-9274-10bf48838a41',
+          'example:test': 'foo'
+        },
+        credentialStatus: {
+          id: 'https://example.com/status/1#67342',
+          type: 'RevocationList2020Status',
+          revocationListCredential: RLC.id
+        }
+      };
+      const documentLoader = 'https://example.com/status/1';
+      let err;
+      let result;
+      try {
+        result = await checkStatus({
+          credential, documentLoader, verifyRevocationListCredential: false});
+      } catch(e) {
+        err = e;
+      }
+      should.not.exist(err);
+      should.exist(result);
+      result.should.be.an('object');
+      result.should.have.property('verified');
+      result.verified.should.be.a('boolean');
+      result.verified.should.be.false;
+      result.should.have.property('error');
+      result.error.should.be.instanceof(TypeError);
+      result.error.message.should.contain(
+        '"documentLoader" must be a function');
+    });
+
+  it('should fail to verify if suite is not an object or array of ' +
+    'objects in checkStatus', async () => {
+    const credential = {
+      '@context': [
+        'https://www.w3.org/2018/credentials/v1',
+        VC_RL_CONTEXT_URL
+      ],
+      id: 'urn:uuid:e74fb1d6-7926-11ea-8e11-10bf48838a41',
+      type: ['VerifiableCredential', 'example:TestCredential'],
+      credentialSubject: {
+        id: 'urn:uuid:011e064e-7927-11ea-8975-10bf48838a41',
+        'example:test': 'bar'
+      },
+      credentialStatus: {
+        id: 'https://example.com/status/1#50000',
+        type: 'RevocationList2020Status',
+        revocationListIndex: '50000',
+        revocationListCredential: RLC.id
+      }
+    };
+    const documentLoader = extendContextLoader(async url => {
+      const doc = documents.get(url);
+      if(doc) {
+        return {
+          contextUrl: null,
+          documentUrl: url,
+          document: doc
+        };
+      }
+      return defaultDocumentLoader(url);
+    });
+    const suite = '{}';
+    let err;
+    let result;
+    try {
+      result = await checkStatus({
+        credential, documentLoader, suite, verifyRevocationListCredential: true
+      });
+    } catch(e) {
+      err = e;
+    }
+    should.not.exist(err);
+    should.exist(result);
+    result.should.be.an('object');
+    result.should.have.property('verified');
+    result.verified.should.be.a('boolean');
+    result.verified.should.be.false;
+    result.should.have.property('error');
+    result.error.should.be.instanceof(TypeError);
+    result.error.message.should.contain(
+      '"suite" must be an object or an array of objects');
+  });
+
+  // FIXME: Need to invoke the error message alone with the verification
```

Comment:
```suggestion
  // FIXME: Need to invoke the error message alone with the verification
```

What does it mean to invoke an error message? Can this FIXME language be more clear? Or, better yet, can we get it addressed in this PR?